numpy/core/src/common/npy_cpu_features.c

```diff
@@ -1,6 +1,5 @@
 #include "npy_cpu_features.h"
 #include "npy_cpu_dispatch.h" // To guarantee the CPU baseline definitions are in scope.
-#include "numpy/npy_common.h" // for NPY_INLINE
```

Comment:
Seems the include was necessary also for `ndarraytypes.h` indirectly.  But then I guess you can also just replace those types instead.  They are around line 454 (only active on x86 apple).

Reply:
Using `ndarraytypes.h` yielded a complaint about using deprecated API ([CI log](https://github.com/numpy/numpy/actions/runs/3558493802/jobs/5977394229)), so I just put `npy_common.h` back.